api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -226,6 +231,36 @@ void shouldProcessClaimWithEligibilityOverrideWithoutNinoAndEmailAddressAndPhone
 
     }
 
+    @Test
+    void shouldProcessClaimWithEligibilityOverrideAndSendTextMessage()
+            throws JsonProcessingException, NotificationClientException {
+        LocalDate expectedDeliveryDate = LocalDate.now().plusWeeks(25);
+        String nino = null;
+        String emailAdress = null;
+        UUID claimId
+                = applyForHealthyStartOverridingEligibilityForAPregnantWomanWithoutNinoAndEmailAndPhone(expectedDeliveryDate, nino, emailAdress, HOMER_MOBILE);
```

Comment:
method name suggests 'WithoutNino' but nino is being passed in even though this is null. Possible rename?